brayns/network/entrypoints/SnapshotEntrypoint.cpp

```diff
@@ -127,9 +127,9 @@ class SnapshotHandler
         // Render
         auto future = brayns::FrameRenderer::asynchronous(*camera, frameBuffer, *renderer, scene);
         const auto msg = "Rendering snapshot ...";
-        while (!ospIsReady(future))
+        while (!future.isReady())
         {
-            const auto percentage = ospGetProgress(future);
+            const auto percentage = future.progress();
             progress.notify(msg, percentage);
             std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
Just noticed but 1s is a lot no ?
For interactive stuff, wouldn't it better 0.1s or even 0.01s ?